pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "svgdigitizer"
+version = "0.11.0"
+license = {text = "GPL-3.0-or-later"}
```

Comment:
@saraedum We can have either the `text`property here or the `file` property. What do you think?

Reply:
Text is better then a file. In particular, `or later` is not part of the license file (it's just the GPL 3 license.)

I am confused that this typing is needed, since here it seems that just spelling out the SPDX identifier should be enough: https://python-poetry.org/docs/pyproject/#license

Reply:
This is what I got without the property:
```
  × Getting requirements to build editable did not run successfully.
  │ exit code: 1
  ╰─> [95 lines of output]
      configuration error: `project.license` must be valid exactly by one definition (2 matches found):
      
          - keys:
              'file': {type: string}
            required: ['file']
          - keys:
              'text': {type: string}
            required: ['text']
      
      DESCRIPTION:
          `Project license <https://peps.python.org/pep-0621/#license>`_.
      
      GIVEN VALUE:
          "GPL-3.0-or-later"
      
      OFFENDING RULE: 'oneOf'
      
      DEFINITION:
          {
              "oneOf": [
                  {
                      "properties": {
                          "file": {
                              "type": "string",
                              "$$description": [
                                  "Relative path to the file (UTF-8) which contains the license for the",
                                  "project."
                              ]
                          }
                      },
                      "required": [
                          "file"
                      ]
                  },
                  {
                      "properties": {
                          "text": {
                              "type": "string",
                              "$$description": [
                                  "The license of the project whose meaning is that of the",
                                  "`License field from the core metadata",
                                  "<https://packaging.python.org/specifications/core-metadata/#license>`_."
                              ]
                          }
                      },
                      "required": [
                          "text"
                      ]
                  }
              ]
          }
      Traceback (most recent call last):
```

Reply:
We are building with setuptools. Should we switch to poetry?

Reply:
No, I just assumed that they had the same syntax.
Sorry for the noise. I guess `text` is fine after all.